src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Publish.targets

```
@@ -121,20 +127,28 @@ Copyright (c) .NET Foundation. All rights reserved.
     ============================================================
                                         _CopyResolvedFilesToPublishPreserveNewest
 
-    Copy _ResolvedFileToPublishPreserveNewest items to the publish directory.
+    Copy _ResolvedFileToPublishPreserveNewest items to the publish directory 
+    (except files that will be bundled, during PublishSingleFile).
     ============================================================
     -->
   <Target Name="_CopyResolvedFilesToPublishPreserveNewest"
           DependsOnTargets="_ComputeResolvedFilesToPublishTypes"
           Inputs="@(_ResolvedFileToPublishPreserveNewest)"
           Outputs="@(_ResolvedFileToPublishPreserveNewest->'$(PublishDir)%(RelativePath)')">
 
+    <ItemGroup>
+      <_ResolvedUnBundledFileToPublishPreserveNewest 
```

```suggestion
      <_ResolvedUnbundledFileToPublishPreserveNewest 
```

nit on casing. Un is not a word on its own.

---

(Obviously, rename all references to it too.)